<?xml version="1.0" encoding="UTF-8"?>
<launch>
  <arg name="manager_name" default="nodelet_manager" />
  <arg name="nodelet_mode" default="standalone" /><!-- set to standalone if you want to use as node-->
  <!-- Nodelet Manager -->
  <group if="$(eval nodelet_mode=='load')">
    <node pkg="nodelet" type="nodelet" name="$(arg manager_name)" args="manager" output="screen"/>
  </group>
  <!-- CAN -->
  <node pkg="nodelet" type="nodelet" name="slcan_bridge" 
  args="$(arg nodelet_mode) can_plugins/SlcanBridge $(arg manager_name)" output="screen"/>

  <!-- <node pkg="nodelet" type="nodelet" name="can_handler" 
    args="$(arg nodelet_mode) can_plugins/CanHandler $(arg manager_name)">
    <param name="beta0" value="$(eval 0x4d8)"/> 
    <remap from="beta/motor0_cmd_vel" to="base/motor0_cmd_vel"/>
  </node> -->
</launch>
